### PR TITLE
Test

### PR DESCRIPTION
**Pull request by Anurodhyadav**
Saved basic user data in database and some authentication as well

### CODE REVIEW

**Review by Drishtantr — APPROVED** [review]

**Comment by shakyaprasen** [comment]
please write better commit names

**Comment by Anurodhyadav** [comment]
> please write better commit names

sorry dai for that.i will make sure i write better commit from now on. Can i merge it,if you approve?

File: src/views/Register.vue [hunk 1]
```diff
@@ -5,11 +5,28 @@
       <h2>Register</h2>
       <p>Please fill in this form to create an account!</p>
       <hr />
+          <div class="form-group">
```
Comment (shakyaprasen) [inline src/views/Register.vue:8]: spacing wrong here. Please use eslint and vetur/prettier for formatting

File: src/views/Register.vue [hunk 2]
```diff
@@ -66,22 +103,36 @@ export default {
...
       try {
-        if (this.email !== '' && this.password !== '') {
+        if (this.email !== '' && this.username !== '' && this.password !== '' && this.password === this.confirmPassword) {
+          this.$vs.loading({
+            type: 'sound',
+          });
           const user = await firebase
             .auth()
             .createUserWithEmailAndPassword(this.email, this.password);
           console.log(user);
```
Comment (shakyaprasen) [inline src/views/Register.vue:120]: console.log here

File: src/views/Register.vue [hunk 3]
```diff
@@ -66,22 +103,36 @@ export default {
...
+          await db.collection('UserData').add({
+            Email: this.email,
```
Comment (shakyaprasen) [inline src/views/Register.vue:122]: use `email` not `Email` and `username` not `Username`

File: src/views/Register.vue [hunk 4]
```diff
@@ -66,22 +103,36 @@ export default {
...
+            Username: this.username,
+          });
+          this.$vs.loading.close();
+          this.acceptAlert('success', 'Registered', 'The account is registered');
           this.$router.replace({ name: 'signin' });
```
Comment (shakyaprasen) [inline src/views/Register.vue:127]: camelcase or dash case for kebabcase i.e. dash case for `signin`

File: src/views/SignIn.vue [hunk 5]
```diff
@@ -77,18 +79,23 @@ export default {
     async SingIn() {
```
Comment (shakyaprasen) [inline src/views/SignIn.vue:79]: camel case for function name

File: src/views/SignIn.vue [hunk 6]
```diff
@@ -77,18 +79,23 @@ export default {
...
       try {
         if (this.email !== '' && this.password !== '') {
+          this.$vs.loading({
+            type: 'sound',
+          });
           const val = await firebase
             .auth()
             .signInWithEmailAndPassword(this.email, this.password);
           console.log(val);
```
Comment (shakyaprasen) [inline src/views/SignIn.vue:88]: console.log

**Review by shakyaprasen — APPROVED** [review]
Fix the issues and merge

File: src/views/Register.vue (hunk 1 above)
Comment (Anurodhyadav) [reply src/views/Register.vue:8]: > Fix the issues and merge

All issues resolved

**Comment by Anurodhyadav** [comment]
Merged with base branch

**Pull request merged**